openapi/kas-fleet-manager-private.yaml

```diff
@@ -19,8 +19,56 @@ servers:
 tags:
   - name: Agent Clusters
     description: ""
+  - name: observatorium-proxy
+    description: Observatorium proxy related endpoints
 
 paths:
+  # observatorium proxy related endpoints
+  '/api/kafkas_mgmt/v1/obs-proxy-verification/{cluster-external-id}':
+    post:
```

Comment:
At the moment we do not have payload, I do not think we'll have any.
I suppose we could just make this a `get`. But I am fine with `post` as well since the operation is an action.